README.md

```diff
@@ -0,0 +1,5 @@
+# ⚡️ QuickJS - A mighty JavaScript engine
+
+Friendly [QuickJS](https://bellard.org/quickjs) fork focused on reigniting the project.
+
+🚧 Work in progress.
```

Comment:
I'll add more stuff here once I have some basic docs inplace.

Reply:
Brevity is a virtue :)